source/dub/dub.d

```diff
@@ -727,17 +783,23 @@ class Dub {
 
 		Params:
 			package_id = Name of the package to be removed
-			location_ = Specifies the location to look for the given package
-				name/version.
+			repoPath = Specifies the location to look for the given package
+				name/version. Relative paths are resolved relative to the
+				current working directory.
 			force_remove = Forces removal of the package, even if untracked
 				files are found in its folder.
 			resolve_version = Callback to select package version.
 	*/
-	void remove(string package_id, PlacementLocation location, bool force_remove,
+	void remove(string package_id, Path repoPath, bool force_remove,
 				scope size_t delegate(in Package[] packages) resolve_version)
 	{
 		enforce(!package_id.empty);
-		if (location == PlacementLocation.local) {
+
+		auto rootNormalized = rootPath;
+		rootNormalized.normalize();
+		repoPath = makeAbsoluteRelativeToCwd(repoPath);
+		repoPath.normalize();
+		if (repoPath == rootNormalized) {
```

Comment:
comparing `Path`s for logical equality (i.e. do they point to the same place) is a bit of a pain. Is there some easy reliable way that I'm missing?